hr-data-generator/src/main/java/uk/gov/gchq/palisade/example/hrdatagenerator/types/Manager.java

```diff
@@ -72,19 +72,19 @@ public void setUid(final UserId uid) {
     }
 
```

Comment:
do we want the @generated for setters and getters?

Reply:
Will do

Reply:
Added to pull/29